code/game/objects/items/devices/PDA/virus_cart.dm

```diff
@@ -66,15 +66,11 @@
 		charges--
 		var/difficulty = 0
 		if(target.cartridge)
-			difficulty += target.cartridge.access_medical
-			difficulty += target.cartridge.access_security
-			difficulty += target.cartridge.access_engine
-			difficulty += target.cartridge.access_clown
-			difficulty += target.cartridge.access_janitor
-			difficulty += target.cartridge.access_manifest * 2
+			difficulty += BitCount(target.cartridge.access&(CART_MEDICAL|CART_SECURITY|CART_ENGINE|CART_CLOWN|CART_JANITOR|CART_MANIFEST))
+			if(target.cartridge.access&CART_MANIFEST) difficulty++ //if cartridge has manifest access it has extra snowflake difficulty
```

Comment:
For something as simple as an iteration the all-in-one-line being more readable point still stands.